packages/veritone-widgets/src/widgets/Table/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { func, string, number, arrayOf, bool, shape, object } from 'prop-types';
+import { Table, PaginatedTable, Column, MenuColumn } from 'veritone-react-common';
+import { omit } from 'lodash';
+import widget from '../../shared/widget';
+
+
+class TableWidget extends React.Component {
+  static propTypes = {
+    data: arrayOf(object).isRequired,
+    columns: arrayOf(shape({
+      dataKey: string.isRequired,
+      header: string,
+      transform: func,
+      menu: bool,
+      onSelectItem: func,
+      cursorPointer: bool,
+      align: string,
+      width: number
+    })),
+    paginate: bool,
+    initialItemsPerPage: number,
+    onCellClick: func,
+    focusedRow: number,
+    renderFocusedRowDetails: func,
+    onShowCellRange: func,
+    onRefreshPageData: func,
+    emptyMessage: string
```

Comment:
i would suggest not enumerating props to a widget except the ones you specifically use in its component -- it's convenient to look at now, but if the table or column components change, we'd have to update both packages to keep things in sync (same reason for spreading props to column/table). 

React component proptypes are the single source of truth.